README.md

```diff
@@ -29,66 +16,73 @@ npm i git-guppy --save-dev
-guppy-hook is like a git-hook preconfigured for specific gulp workflows.
+gulp.task('pre-commit', ['unit'], function () {
+  // your pre-commit task's code.
+});
+```
 
-You can install *guppy-hooks* via `npm` just like any other package. For every valid
-git-hook name, there exists a "guppy-[hookname]" package that automatically installs
-the related hook to your `.git/hooks` dir, e.g. "guppy-pre-commit" or "guppy-post-update".
-Just add the guppy-hook you need as a dev-dependency in your project.
+### 2. Installing git-hooks
 
-Search ["guppy-hook" on npm](https://www.npmjs.com/search?q=guppy-hook) to find all 
-guppy-hook packages. Or run `npm search guppy-hook` from the commandline.
+To define that your project implements a git-hook you can simply add the hook's
+name to the introduced *package.json*'s property *guppy-hooks*, which should
+always be an array such as follows:
 
-### gulp integration
+```json
+{
+  "guppy-hooks": [
+    "pre-commit",
+    "post-commit"
+  ]
+}
+```
 
-> :warning: **Stop!** If you are using a guppy-hook package, refer to the 
-documentation for that package. You do not need the steps below unless you are 
-adding custom guppy integration to your gulpfile or authoring your own guppy-hook
-package.
+After that, you can install *git-guppy* as a dependency of your project
+(`npm install git-guppy --save-dev`) and have it's *postinstall* script automatically create the proper hook files inside *.git/hooks* directory.
 
-guppy exposes a few simple methods to help you superpower your git-hooks with 
-gulp tasks.
+> If you already had *git-guppy* installed as a dependency of your project
+before this step, make sure to remove the package from the node_modules
+directory and running `npm install` once again so to force NPM to run
+*git-guppy*'s postinstall script, whcih is responsible for creating the hook
+files mentioned above.
 
-Before you dive in, initialize guppy by passing in your gulp reference:
 
-```js
-var gulp = require('gulp');
-var guppy = require('git-guppy')(gulp);
-```
+## Advanced usage
 
-Then simply define some gulp tasks in your `gulpfile.js` whose names match 
-whichever git-hooks you want to be triggerable by git.
+### gulp integration
+
+guppy exposes a few simple methods to help you superpower your git-hooks with
+gulp tasks. This is useful when you need to access some meta information from
+a git process - such as files being commited, or the commiting message.
+
+To access these methods, you need to initialize guppy by passing in your gulp
+reference as follows:
 
 ```js
-gulp.task('pre-commit', function () {
-  // see below
-});
+var gulp = require('gulp');
+var guppy = require('git-guppy')(gulp);
 ```
 
-*Note: if you are working directly with guppy rather than installing a guppy-hook
-you will need to manually install the associated git-hooks using the 
-[guppy-cli](https://github.com/therealklanni/guppy-cli) commandline tool.*
-
```

Comment:
**Motivation**: this is not really necessary, with the new hook installing method. We should, though, find a place to mention the existence of the cli tool.